core/src/packets/types.rs

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright 2019 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//! Custom primitive wrapper types for converting data to/from network byte
+//! order.
+
+use std::convert::From;
+use std::fmt;
+use std::ops;
+
+/// Big-endian type for `u16`.
+///
+/// Used to convert packet fields to host byte order on get and network byte
+/// order on set.
+#[allow(non_camel_case_types)]
+#[derive(Clone, Copy, Debug, Default, Hash, PartialEq, Eq, PartialOrd, Ord)]
+#[repr(C, packed)]
+pub struct u16be(pub u16);
+
+impl u16be {
+    /// The 16-bit unsigned integer in big-endian order.
```

Comment:
I think you made this change in the wrong place. this is for the main type description above. description here should be,

> The smallest value that can be represented by this integer type.

Reply:
derp.